### PR TITLE
add Get-AzVmssVmRunCommand on SDK based cmdlet 

### PR DESCRIPTION
**Pull request by grizzlytheodore**
## Description
This is mirroring what we already did to Get-AzVMRunCommand in the last release: https://github.com/Azure/azure-powershell/pull/19272

Get-AzVmssVMRunCommand was originally autorest.powershell generated. But I am removing that from compute.autorest in this pr:
https://github.com/Azure/azure-powershell/pull/19826

We are moving it to cmdlet built on SDK because we wanted to achieve two things:
Have all the fields show as a list format when we get back list of RunCommands
Have instanceView only show when -Expand InstanceView

So to achieve these customizations I moved it to cmdlet built on SDK here.

There is no breaking change here because currently from the documentation https://learn.microsoft.com/en-us/powershell/module/az.compute/get-azvmssvmruncommand?view=azps-9.0.0, 1st, 2nd, and 5th(last) parameter sets arent working

And the one parameter set we are adding in this PR is a combination of 3rd and 4th parameter sets by having -RunCommandName as optional.

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1060","The parameter set 'List' for cmdlet 'Get-AzVmssVMRunCommand' is no longer the default parameter set.","Change the default parameter for cmdlet 'Get-AzVmssVMRunCommand' back to 'List'."
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1060","The parameter set 'List' for cmdlet 'Get-AzVmssVMRunCommand' is no longer the default parameter set.","Change the default parameter for cmdlet 'Get-AzVmssVMRunCommand' back to 'List'."
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1060","The parameter set 'List' for cmdlet 'Get-AzVmssVMRunCommand' is no longer the default parameter set.","Change the default parameter for cmdlet 'Get-AzVmssVMRunCommand' back to 'List'."
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","2000","The cmdlet 'Get-AzVmssVMRunCommand' no longer supports the parameter 'SubscriptionId' and no alias was found for the original parameter name.","Add the parameter 'SubscriptionId' back to the cmdlet 'Get-AzVmssVMRunCommand', or add an alias to the original parameter name."
```
Comment (VeryEarly) [inline tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv:10]: this is breaking change

Comment (grizzlytheodore) [reply tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv:10]: wrote in the description above, but that parameter set was not working to begin with

File: tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv [hunk 2]
```diff
@@ -0,0 +1,23 @@
+"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
+"Az.Compute","Get-AzVmssVMRunCommand","Get-AzVmssVMRunCommand","0","1020","The cmdlet 'Get-AzVmssVMRunCommand' no longer has output type 'Microsoft.Azure.PowerShell.Cmdlets.Compute.Models.Api20210701.IVirtualMachineRunCommand'.","Make cmdlet 'Get-AzVmssVMRunCommand' return type 'Microsoft.Azure.PowerShell.Cmdlets.Compute.Models.Api20210701.IVirtualMachineRunCommand'."
```
[the remark this answers is not in the document]
Comment (grizzlytheodore) [reply tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv:2]: see IVirtualMachineRunCommand here: 
https://github.com/Azure/azure-powershell/blob/6bf24588c1e938c49080ad99f4d9411667b6c740/src/Compute/Compute.Autorest/generated/api/Models/Api20210701/VirtualMachineRunCommand.cs#L340

see PSVirtualMachineRunCommand here:
 https://github.com/Azure/azure-powershell/blob/a81b266b8a4e0f7e39b2a8cc164e3895e8f85634/src/Compute/Compute/Manual/PSVirtualMachineRunCommand.cs#L9

all the properties match

**Comment by grizzlytheodore** [comment]
please see my responses @VeryEarly

File: tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv (hunk 1 above)
Comment (VeryEarly) [reply tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv:10]: please try set VirtualMachineScaleSetVMRunCommandsClient.SubscriptionId
before this line: 
https://github.com/Azure/azure-powershell/blob/a81b266b8a4e0f7e39b2a8cc164e3895e8f85634/src/Compute/Compute/VirtualMachine/RunCommand/GetAzVmssVmRunCommand.cs#L75

File: tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv (hunk 2 above)
[the remark this answers is not in the document]
Comment (VeryEarly) [reply tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv:2]: these two objects do not match, please check carefully, properties in IVirtualMachineRunCommand are flattened

[the remark this answers is not in the document]
Comment (grizzlytheodore) [reply tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv:2]: also @dingmeng-xue  is aware of this migration, but here is the reasonings I sent him via email: 
1.	The feature owner wanted the returned output as list format.
a.	Even when we specify -RunCommandName to retrieve 1 RunCommand resource, it is returned in table format. The having ~10 properties that the user needed to see, the table format did not provide all the information needed for the users due to the PowerShell window size constraints. 
2.	Have instanceView properties only shown when -Expand is provided.
a.	The output was always showing “IntanceView” properties. So when “-Expand InstanceView” was not provided, those properties were always empty.

[the remark this answers is not in the document]
Comment (grizzlytheodore) [reply tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv:2]: and 2a is the result of having a flat hierchy

[the remark this answers is not in the document]
Comment (VeryEarly) [reply tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv:2]: got it, breaking changes on broken cmdlets won't be considered as blocker.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**